doc/overview/consensus/index.rst

```diff
@@ -0,0 +1,86 @@
+Consensus Protocols
+===================
+
+CCF supports multiple consensus protocols.
+
+The default consensus protocol for CCF is Crash Fault Tolerance (:term:`CFT`) and there is an experimental option of enabling Byzantine Fault Tolerance (:term:`BFT`).
+
+Below, we give an overview over the nodes state machine in both settings and the retirement mechanics that apply across the two protocols. 
+
+CFT Consensus Protocol
+-----------------------
+
+The crash fault tolerant implementation in CCF is based on Raft. You can find more information on the Raft implementation in CCF :doc:`here <cft>`.
+
+CFT parameters can be configured when starting up a network (see :doc:`here </operations/start_network>`). The parameters that can be set via the CLI are:
+
+- ``raft-timeout-ms`` is the Raft heartbeat timeout in milliseconds. The Raft leader sends heartbeats to its followers at regular intervals defined by this timeout. This should be set to a significantly lower value than ``--raft-election-timeout-ms``.
+- ``raft-election-timeout-ms`` is the Raft election timeout in milliseconds. If a follower does not receive any heartbeat from the leader after this timeout, the follower triggers a new election.
+
+BFT Consensus Protocol
+----------------------
+.. warning:: CCF with BFT is currently still in development and should not be used in a production environment.
+
+BFT is still under development and should not be enabled in a production environment. More details on this mode is given :doc:`here <bft>`. There is an open research question of `node identity with Byzantine nodes <https://github.com/microsoft/CCF/issues/893>`_.
+
+By default CCF runs with CFT. To run CCF with BFT the ``--consensus bft`` CLI argument must be provided when starting up the nodes (see :doc:`/operations/start_network` for starting up a network and nodes).
```

Comment:
This is explicitly forbidden/disabled when running a release version of CCF so we may want to mention that it's only when CCF is built from source (link to https://microsoft.github.io/CCF/main/contribute/build_ccf.html).

Reply:
Clarified in cc4fff8